engine/types.asm

```diff
@@ -91,9 +91,32 @@ GetTypeName: ; 50964
 	ld h, [hl]
 	ld l, a
 	ld de, StringBuffer1
-	ld bc, 13
+	ld bc, MOVE_NAME_LENGTH
 	jp CopyBytes
 ; 5097b
 
 
 INCLUDE "data/types/names.asm"
+
+
+Unreferenced_GetGen1TrainerClassName: ; 50a28
```

Comment:
what? why is this here?

Reply:
Should it go in engine/routines/unreferenced_getgen1trainerclassname.asm?